src/main/scala/lms/thirdparty/cuda/cuda.scala

```diff
@@ -700,7 +701,7 @@ trait CudaOps extends Dsl with StackArrayOps with SizeTOps with CLibs with CudaF
   def threadIdxX: Rep[Int] = cmacro[Int]("threadIdx.x")
   def threadIdxY: Rep[Int] = cmacro[Int]("threadIdx.y")
   def threadIdxZ: Rep[Int] = cmacro[Int]("threadIdx.z")
-
+  def blockRows: Rep[Int] = cmacro[Int]("BLOCK_ROWS")
```

Comment:
I think `BLOCK_ROWS` is a local macro defined in their code, not a macro we can just use :)

Reply:
added it as a constant into our code